packages/config/src/config.js

```diff
@@ -34,6 +34,7 @@ module.exports = ({
         console.warn('betaEnv is deprecated in favor of env');
     }
 
+    const devServerPort = typeof port === 'number' ? port : useProxy || standalone ? 1337 : 8002;
```

Comment:
Would it be possible to wrap `: (useProxy || standalone) ? ...` so the priority is clear?